technical/indicators.py

```diff
@@ -785,9 +785,11 @@ def vpci(dataframe, period_short=5, period_long=20):
     return vpci
 
 
-def williams_percent(dataframe):
-    from pyti.williams_percent_r import williams_percent_r
-    return williams_percent_r(dataframe['close'])
+def williams_percent(dataframe, period=14, colname='close'):
```

Comment:
LGTM 👍 , brilliant, we should replace other appearances of pyti with pure vectorized pandas calculations...

with one note: other functions use `field='close'` parameter to specify the column with the price, on which the indicator is calculated; should be made same way here

Reply:
Changed it by myself